conda_build/main_develop.py

```diff
@@ -32,6 +36,12 @@ def main():
         nargs='+',
         help="Path to the source directory."
     )
+    p.add_argument(
+                   '-l', '--post-link',
```

Comment:
Ah, I see. In that case, the option name should probably just be called `--no-pth`.